tests/test_gpu.py

```diff
@@ -11,8 +11,8 @@ def test_gpu(factory, client, org_idx, default_datasets, workers):
     nvidia_drivers = "nvidiacuda11.8.0-base-ubuntu22.04"
 
     # Need the base image, the minimal image does not have pip
-    dockerfile = f"""
-FROM ghcr.io/substra/substra-tools:latest-{nvidia_drivers}-python3.7
+    dockerfile = """
+FROM python:3.12-slim
```

Comment:
Isn't that slightly anticipated? 
```suggestion
FROM python:3.11-slim
```

Reply:
Indeed, good catch